snow/engine/snowman/syncer/state_syncer_test.go

```diff
@@ -211,7 +216,7 @@ func TestStateSyncNotFoundOngoingSummaryIsNotIncludedAmongFrontiers(t *testing.T
 	}
 
 	// Connect enough stake to start syncer
-	for nodeID := range vdrs.Map() {
+	for nodeID := range vdrs.GetMap(ctx.SubnetID) {
```

Comment:
Though this test we can replace `GetMap` with `GetValidatorIDs(subnetID ids.ID)`.

Reply:
done